exercises/sum-of-multiples/.meta/solutions/sum_of_multiples.rb

```diff
@@ -1,10 +1,14 @@
+module BookKeeping
+  VERSION = 1
+end
+
 class SumOfMultiples
   attr_reader :multiples
   def initialize(*multiples)
     @multiples = multiples
   end
 
-  def to(limit)
+  def take(limit)
```

Comment:
Although I had forgotten that the limit was excluded, so the name change may be inappropriate.